starters/vue-starter-ts/.env.example

```diff
@@ -2,7 +2,7 @@
 # PCC_HOST=https://pcc-gfttxsojwq-uc.a.run.app
 
 # The id which can be retrieved from the add-on or CLI
-PCC_SITE_ID=12345
```

Comment:
I wonder if we should rewrite this in nuxt.config.js instead of changing it here so that the documentation/instructions for env variables doesn't have to be different for vue? Changing the env name also requires changing the CLI; preferably we don't have to do that.

Reply:
I agree, updated how the variables are added to the app 👍